### DISCUSSION TITLE
Azuracast performance

### DISCUSSION THREAD

**Discussion by astra-fm** [AzuraCast/AzuraCast#6345] [Q&A]
Hello, on one hand, I have AzuraCast installed on DigitalOcean. I've contracted a droplet with the following information:
8 GB Memory / 4 Intel vCPUs / 160 GB Disk / FRA1 - AzuraCast 0.17.7 on Ubuntu 22.04.
The CPU is always at 100%.

On the other hand, I have a website that displays the radio's information, mainly making calls to the AzuraCast API to show the artist's information, etc...
The problem is, the information on the page that makes the call to the API takes a very long time to load, and I'm not sure if it's a problem with the AzuraCast API, the resources AzuraCast consumes, or the server where I have AzuraCast set up.
Could someone who has experienced a similar situation guide me towards a solution?

This is my website: https://media.studioxplore.com

**Comment by scysys** [comment]
First you should check with tools like "htop" where you have spikes.

**Reply by astra-fm** [reply]
Hi scysys,
Thanks for your help.
this is what I have in htop
<img width="1033" alt="htop" src="https://github.com/AzuraCast/AzuraCast/assets/136566649/660fd0e9-8e06-46b3-b4e4-224b6226e74c">
